serde_derive/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "serde_derive"
+version = "0.8.4"
+authors = ["Erick Tryzelaar <erick.tryzelaar@gmail.com>"]
+license = "MIT/Apache-2.0"
+description = "Macros 1.1 implementation of #[derive(Serialize, Deserialize)]"
+homepage = "https://serde.rs"
+repository = "https://github.com/serde-rs/serde"
+documentation = "https://serde.rs/codegen.html"
+keywords = ["serde", "serialization"]
+include = ["Cargo.toml", "src/**/*.rs"]
+
+[lib]
+name = "serde_derive"
+rustc-macro = true
+
+[dependencies]
+serde_codegen = { version = "=0.8.5", path = "../serde_codegen" }
```

Comment:
Same reason we pin `serde_macros`. See https://github.com/serde-rs/serde/issues/470#issuecomment-236307729. It allows you to use an old nightly with an old serde_derive without pulling in a new serde_codegen that no longer compiles on the old nightly.